test/networks/views/bfs_topo_view.cpp

```diff
@@ -0,0 +1,141 @@
+//
+// Created by benjamin on 08.04.24.
+//
+#include <catch2/catch_test_macros.hpp>
+
+#include <fiction/networks/views/bfs_topo_view.hpp>
+
+#include <mockturtle/networks/aig.hpp>
+#include <mockturtle/traits.hpp>
+
+#include <cstdint>
+#include <set>
+#include <vector>
+
+using namespace fiction;
+
```

Comment:
The traits of the new view should be tested as well. See the test cases for layered layout types, e.g., https://github.com/cda-tum/fiction/blob/main/test/layouts/obstruction_layout.cpp#L19-L96